src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -98,9 +98,7 @@ export default class MutationMapper extends React.Component<IMutationMapperProps
                             />
                             <div style={{marginLeft:"45px", marginTop:"5px", marginBottom:"10px"}}>
                                 <ButtonGroup className="pull-right">
-                                    <Button className="btn-default" disabled={this.props.store.pdbChainDataStore.allData.length === 0} onClick={this.toggle3dPanel}>
-                                        3D Structure »
-                                    </Button>
+                                    <Button className="btn-default" disabled={this.props.store.pdbChainDataStore.allData.length === 0} onClick={this.toggle3dPanel} dangerouslySetInnerHTML={{ __html:['3D Structure &','raquo;'].join('') }} />
```

Comment:
@onursumer done. We actually used the same thing here: https://github.com/alisman/cbioportal-frontend/commit/5f2280fc4f721775f2715358dfaf9b1d78124b56. But yeah I agree that using awesome font is a lot easier to read